aiven/client/argx.py

```diff
@@ -27,6 +28,11 @@
 except ImportError:
     __version__ = "UNKNOWN"
 
+# cached_property only exists since python 3.8
+SKIP_EVALUATION_TYPES = (property, )
+if hasattr(functools, "cached_property"):
+    SKIP_EVALUATION_TYPES += (functools.cached_property, )
```

Comment:
Mutating tuples feels a bit dirty, but python has no problem with it so i guess fine.
A list here would be preferable, but that's a minor thing and shouldn't hold this off.

Or, you could:
```
if hasattr(functools, "cached_property"):
    SKIP_EVALUATION_TYPES = (property, functools.cached_property)
else:
    SKIP_EVALUATION_TYPES = (property,)
```

All in all, really unimportant :)